NEWS.txt

```diff
@@ -3,249 +3,183 @@ Release Notes
- - Support for ``MaxAndArgMax`` for some axis combination
- - Support for solve (using cusolver), erfinv and erfcinv
- - Implemented ``GpuAdvancedSubtensor``
+ - Prevent GPU initialization when not required
+ - Added disk caching option for kernels
+ - Added method ``my_theano_function.sync_shared()`` to help synchronize GPU Theano functions
+ - Added useful stats for GPU in profile mode
+ - Added Cholesky op based on ``cusolver`` backend
+ - Added GPU ops based on `magma library <http://icl.cs.utk.edu/magma/software/>`_:
+   SVD, matrix inverse, QR, cholesky and eigh
+ - Added ``GpuAdvancedIncSubtensor``
+ - Added ``GpuCublasTriangularSolve``
+ - Added atomic addition and exchange for ``long long`` values in ``GpuAdvancedIncSubtensor1_dev20``
+ - Fixed C code for log gamma function, now supporting all types except complex types.
+ - Support GPU SoftMax in both OpenCL and CUDA
+ - Support offset parameter ``k`` for ``GpuEye``
+ - ``CrossentropyCategorical1Hot`` and its gradient are now lifted to GPU
+
+ - Better cuDNN support
+
+   - Official support for versions >= ``v5``
+   - Better support and loading on Windows and Mac
+   - Support cuDNN v6 dilated convolutions
+   - Support cuDNN v6 reductions
+   - Added new theano flags ``cuda.include_path``, ``dnn.base_path`` and ``dnn.bin_path``
+     to help configure Theano when CUDA and cuDNN can not be found automatically.
+
+ - Updated ``float16`` support
+
+   - Added documentation for GPU float16 ops
+   - Support ``float16`` for ``GpuGemmBatch``
+   - Started to avoid lifting ``float16`` computations that are not supported on GPU
 
 New features:
- - ``OpFromGraph`` now allows gradient overriding for every input
- - Added Abstract Ops for batch normalization that use cuDNN when available and pure Theano CPU/GPU alternatives otherwise
- - Added gradient of solve, tensorinv (CPU), tensorsolve (CPU), searchsorted (CPU), DownsampleFactorMaxGradGrad (CPU)
- - Added Multinomial Without Replacement
- - Allowed partial evaluation of compiled function
- - More Rop support
- - Indexing support ellipsis: ``a[..., 3]```, ``a[1,...,3]``
- - Added ``theano.tensor.{tensor5,dtensor5, ...}``
- - compiledir_format support device
- - Added New Theano flag ``conv.assert_shape`` to check user-provided shapes at runtime (for debugging)
- - Added new Theano flag ``cmodule.age_thresh_use``
- - Added new Theano flag ``cuda.enabled``
- - Added new Theano flag ``nvcc.cudafe`` to enable faster compilation and import with old CUDA back-end
- - Added new Theano flag ``print_global_stats`` to print some global statistics (time spent) at the end
- - Added new Theano flag ``profiling.ignore_first_call``, useful to profile the new gpu back-end
- - remove ProfileMode (use Theano flag ``profile=True`` instead)
-
+ - Added a wrapper for `Baidu's CTC <https://github.com/baidu-research/warp-ctc>`_ cost and gradient functions
+ - Added scalar and elemwise ops for modified Bessel function of order 0 and 1 from ``scipy.special``
```

Comment:
specify that we added it on CPU only. At least, this is what I recall, double check.